pkg/image/prune/imagepruner.go

```diff
@@ -190,6 +190,7 @@ func addImageStreamsToGraph(g graph.Graph, streams *imageapi.ImageStreamList, al
 
 		glog.V(4).Infof("Adding ImageStream %s/%s to graph", stream.Namespace, stream.Name)
 		isNode := imagegraph.EnsureImageStreamNode(g, stream)
+		imagegraph.EnsureAllImageStreamTagsNode(g, stream)
```

Comment:
This isn't needed by pruning, but since I know you want to have consistent ways to build the nodes and edges...

Wouldn't it make more sense to make `EnsureAllImageStreamTagsNode` private and call it from within `EnsureImageStreamNode`?